### PR TITLE
Make navigation work

### PR DESCRIPTION
**Pull request by sagesmith-wf**
#### Changes:
- Broke out constants into their own files
- Fixed broken unit tests
- Placed agent
- Began work on navigation

### CODE REVIEW

File: src/actions.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from directions import NORTH, SOUTH, EAST, WEST
+
+"""
+ Actions
+ """
+MOV_N = NORTH
+MOV_E = EAST
+MOV_S = SOUTH
+MOV_W = WEST
```
Comment (Kuroshii) [inline src/actions.py]: There is no action for "move north", the agent can only way an agent can move is by rotating left or right and moving forward.

Comment (sagersmith8) [reply src/actions.py]: Noted

Comment (sagersmith8) [reply src/actions.py]: Comment addressed

**Comment by Kuroshii** [comment]
I've dealt with some stuff a little differently (mostly enums); this isn't strictly wrong, but I like how I've done it a little more.

**Review by JWhitney406 — APPROVED** [review]
Looks good!

**Pull request merged**